js/modules/k6/experimental/tracing/goja.go

```diff
@@ -0,0 +1,11 @@
+package tracing
+
+import "github.com/dop251/goja"
+
+// isNullish checks if the given value is nullish, i.e. nil, undefined or null.
+//
+// This helper function emulates the behavior of Javascript's nullish coalescing
+// operator (??).
+func isNullish(value goja.Value) bool {
+	return value == nil || goja.IsUndefined(value) || goja.IsNull(value)
+}
```

Comment:
Should we move to `js/common.go`?

Reply:
That's a good idea 🤔 What do you think @mstoykov ?

Reply:
I am okay with that, but not certain if that shouldn't be done in a separate PR and also use it across the codebase - which will make it bigger. 

IMO we can do that as a refactoring PR in the cooldown period.

Reply:
Let's do it this way indeed then 👍🏻 I took note to do that during the cooldown 🙇🏻